scripts/dockerfiles/rsyslog_conf/etc/logrotate.d/rsyslog

```diff
@@ -1,14 +1,13 @@
 /var/log/syslog
 {
 	rotate 7
-	size 5M
+	maxsize=10M
+	minsize=9M
 	missingok
 	notifempty
 	delaycompress
 	compress
-	postrotate
-		reload rsyslog >/dev/null 2>&1 || true
-	endscript
+	copytruncate
```

Comment:
Yeah, the reason to go with `copytruncate` is that we've seen it work for everything else.